src/app/containers/Ad/Amp/index.jsx

```diff
@@ -85,50 +86,99 @@ export const AMP_ACCESS_FETCH = service => {
   );
 };
 
+const AdContent = props => {
+  // eslint-disable-next-line react/prop-types
+  const { script, service, dir, label, slotType, pageType } = props;
+  return (
+    <>
+      <StyledLink
+        href={LABEL_LINK}
+        script={script}
+        service={service}
+        dir={dir}
+        tabIndex="-1"
+      >
+        {label}
+      </StyledLink>
+      <AdSlot service={service} slotType={slotType} pageType={pageType} />
+    </>
+  );
+};
+
+const AdWithoutPlaceholder = props => {
+  // eslint-disable-next-line react/prop-types
+  const { ariaLabel, slotType } = props;
+  return (
+    <DisplayWrapper amp-access="toggles.ads.enabled" amp-access-hide="true">
+      <AdSection
+        aria-label={ariaLabel}
+        role="region"
+        data-e2e="advertisement"
+        aria-hidden="true"
+      >
+        <AdContainer slotType={slotType}>
+          <StyledWrapper>
+            <AdContent {...props} />
+          </StyledWrapper>
+        </AdContainer>
+      </AdSection>
+    </DisplayWrapper>
+  );
+};
+
+const AdWithPlaceholder = props => {
+  // eslint-disable-next-line react/prop-types
+  const { ariaLabel, slotType } = props;
+  return (
+    <AdSection
+      aria-label={ariaLabel}
+      role="region"
+      data-e2e="advertisement"
+      aria-hidden="true"
+    >
+      <AdContainer slotType={slotType}>
+        <StyledWrapper>
+          <DisplayWrapper
+            amp-access="toggles.ads.enabled"
+            amp-access-hide="true"
+          >
+            <AdContent {...props} />
+          </DisplayWrapper>
+        </StyledWrapper>
+      </AdContainer>
+    </AdSection>
+  );
+};
+
 const AmpAd = ({ slotType }) => {
   const { translations, dir, script, service } = useContext(ServiceContext);
   const { pageType } = useContext(RequestContext);
+  const { enabled: showPlaceholder } = useToggle('adPlaceholder');
```

Comment:
Is there any reason you decided to use a toggle here rather than defining this in the service configs? I just can't picture a scenario where we would choose to toggle this on/off out of hours etc. Have you tested what happens when the toggle is unavailable, what would the default option be? I'd imagine not showing the placeholder might be best as a default?

Reply:
Use of a toggle arose from a chat with Gavin and Jane. I can change it to use the service configs, no bother

Reply:
Updated to the service configs